### PR TITLE
Ts dev

### PR DESCRIPTION
**Pull request by dron0gus**
Remove useless command support, simplify...
Implement Full Optimized – High Speed Scatter mode.

### CODE REVIEW

File: firmware/ini/rusefi_wb_f1_dual.ini [hunk 1]
```diff
@@ -30,15 +30,14 @@ enable2ndByteCanID = false
...
-   nPages              = 1
-
-   pageIdentifier      = "\x00\x00"
-   pageReadCommand     = "R%2o%2c"
-   burnCommand         = "B"
-   pageActivate        = "P"
-   pageValueWrite      = "W%2o%v"
-   pageChunkWrite      = "C%2o%2c%v"
-   crc32CheckCommand   = "k%2o%2c"
+   nPages              = 2
```
Comment (mck1117) [inline firmware/ini/rusefi_wb_f1_dual.ini:33]: why two pages?

Comment (dron0gus) [reply firmware/ini/rusefi_wb_f1_dual.ini:33]: For `scatteredOchGetCommand` and it's `highSpeedOffsets`

**Comment by dron0gus** [comment]
From this:
Screenshot from 2022-12-19 22-28-16
To this:
Screenshot from 2022-12-20 01-46-12
UART@115200.

**Comment by dron0gus** [comment]
TS is slooooow

**Comment by dron0gus** [comment]
`echo 0 > /sys/bus/usb-serial/devices/ttyUSB0/latency_timer` improves the situation.
Screenshot from 2022-12-20 02-16-49

**Comment by dron0gus** [comment]
@mck1117 any concerns? Merge?

**Comment by dron0gus** [comment]
@mck1117 ping?

**Pull request merged**